backend_addon/{{ cookiecutter.__folder_name }}/Makefile

```diff
@@ -45,23 +45,22 @@ all: build
 help: ## This help message
 	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
-$(BIN_FOLDER)/pip $(BIN_FOLDER)/tox $(BIN_FOLDER)/pipx $(BIN_FOLDER)/uv $(BIN_FOLDER)/mxdev:
+$(BIN_FOLDER)/pip $(BIN_FOLDER)/uv:
 	@echo "$(GREEN)==> Setup Virtual Env$(RESET)"
 	$(PYTHON) -m venv $(VENV_FOLDER)
-	$(BIN_FOLDER)/pip install -U "pip" "uv" "wheel" "pipx" "tox" "pre-commit"
-	if [ -d $(GIT_FOLDER) ]; then $(BIN_FOLDER)/pre-commit install; else echo "$(RED) Not installing pre-commit$(RESET)";fi
+	$(BIN_FOLDER)/pip install -U "pip" "uv"
```

Comment:
Are you sure we don't need `wheel`?

Reply:
I've tested it and it seems we do not need it (Even tried to build the package with uv and it worked)